### ISSUE TITLE
Is it needed to somehow turn off Dropblock layer during inference time?

### ISSUE THREAD

**Issue by Westerby** [miguelvr/dropblock#33]
Or does it turn off by default?

**Comment by miguelvr** [comment]
you just need to set the module in evaluation mode beforehand.

e.g

```python
import torch
from dropblock import DropBlock2D

# (bsize, n_feats, height, width)
x = torch.rand(100, 10, 16, 16)

model = nn.Sequential(
    [
        nn.Linear(10),
        DropBlock2D(block_size=3, drop_prob=0.3)
    ]
)

model.eval()

print(model(x))
```
